pythran/tables.py

```diff
@@ -4424,6 +4414,27 @@ def partialsum(seq):
     del MODULES['operator_']['matmul']
     del MODULES['operator_']['__matmul__']
 
+# OMP version
```

Comment:
why do you need to do that instead of the previous scanism mechanism?

Reply:
? I don't remember, but some module use the table for getting omp function then fail on Windows because the function doesn't existe on omp 2.0

Reply:
Yes msvc omp support is 2.0, not more